docs/governance/how-do-i-participate-in-governance.md

```diff
@@ -128,6 +131,9 @@ octez-client transfer 0 from tz1RfbwbXjE8UaRLLjZjUyxbj4KCxibTp9xN to KT1HfJb718f
   --arg "\"sr1EStimadnRRA3vnjpWV1RwNAsDbM3JaDt6\""
 ```
 
+After the new kernel becomes active, bakers must upgrade their nodes to it to continue to participate in Etherlink.
```

Comment:
I’m not sure how, but we should reword here. We don’t need to upgrade nodes (as in, upgrading to a more recent version), we need to provision them with the correct preimages.

Reply:
How about this?

> After the new kernel becomes active, bakers must provide their nodes with the preimages for the new kernel to continue to participate in Etherlink.

Reply:
s/provide/provision?

Reply:
What do you mean by "provision?" Do you mean that node operators would provision a new container or virtual image? I assumed that node operators could merely restart the node process when the preimages endpoint gets updated.

Reply:
I’m using “provision” here without any specific technological implication, but rather as a way to say “make something available to”.

Here it’s simply copying the correct the files in the expected directory, no need to restart the node.

Anyway, I think your suggestion works as well.

Reply:
Updated to:

> After the new kernel becomes active, bakers must provide their nodes with the preimages for the new kernel to continue to participate in Etherlink. They can copy the new preimages into the node data directory without stopping or restarting the node. If a node uses a preimages endpoint as described in [Running an Etherlink EVM node](http://localhost:3000/network/evm-nodes) and [Running an Etherlink Smart Rollup node](http://localhost:3000/network/smart-rollup-nodes), the node updates automatically.